server/sandbox.go

```diff
@@ -172,6 +172,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		if err != nil {
 			return nil, err
 		}
+		g.SetProcessSelinuxLabel(processLabel)
 	}
```

Comment:
I am not sure we need it here.  But we need to Reserve the label when OCID reads the existing containers.

Yes, I forgot about that one.  We need to release it also.
label.UnreserveLabel(processLabel)

On Delete